content/datenschutz/_index.md

```diff
@@ -0,0 +1,506 @@
+---
```

Comment:
Theoretisch könnte diese Datei einfach `datenschutz.md` heißen und müsste dann nicht in einen Extra-Ordner. Aktuell erwartet Hugo nämlich aufgrund des Namensschemas (`_index.md`), dass es hier noch mehr Unterseiten gibt. Aber diese Seite ist ja die einzige zu Datenschutz, die wir haben.